src/pages/RiddleDetailPage.tsx

```diff
@@ -18,11 +17,15 @@ const RiddleDetailPage: FC = () => {
 		const loadAndSetRiddle = async () => {
```

Comment:
Funkce `loadAndSetRiddle()` zde i v souboru `RiddeSolvingPage.tsx` by se dala přesunout do vlastních hooků. Možná i do jednoho? Zde se v rámci zmíněné funkce volá fetchovací funkce `fetchRiddleSimpleDetail()` a v druhém souboru `fetchRiddleComplexDetail()`.

Reply:
Souhlasím, taky jsem nad tím už uvažoval 😃 Všiml jsem si, že jsou až na volané funkce a typy stavu identické, ale upřímně nevím, jestli udělat tam nějaký if podle parametru výrazně nesniží čitelnost, takže za mě asi lepší mít dva čitelné hooky než jeden parametrizovaný, i když mě ten duplikovaný kód taky dráždí.
PS: Nechápu, jak někdo může používat čistý JS bez TS, to by byly v našem případě každou chvíli errory, že někde něco neexistuje nebo je undefined 🗡️

Reply:
Ok, tedy udělám dva hooky! A if zkusím vyhodit někam do separátní funkce! :)